poutyne/framework/model.py

```diff
@@ -717,10 +748,23 @@ def cuda(self, *args, **kwargs):
         method. Notice that the device is saved so that the batches can send to the right device
         before passing it to the network.
 
+        Note:
+            PyTorch optimizers assume that the parameters have been transfered to the right device
+            before their creations. Furthermore, future versions of PyTorch won't modify in-place
+            anymore the parameters of a PyTorch module when transfering them to another device.
```

Comment:
Would probably reword a little bit here. What about

"Furthermore, future versions of PyTorch will no longer modify the parameters of a PyTorch module in-place when transferring them to another device."

There was also a typo in "transfering".

Reply:
We should also include at least one citation. #Science